src/views/TeamsPermissions/EditView.vue

```diff
@@ -54,6 +56,13 @@
       required: true
     }
   })
+
+  const router = useRouter()
+
+  const handleLoadFail = () => {
+    router.push({ name: 'teams-permission' })
```

Comment:
add an E2e test cenário, where this user, without the permission will be pushed to  teams-permission route  when Edit fails.

You can create a e2e test, and intercept de load service to create a error and ensure this behavior